src/test/fuzz/coins_view.cpp

```diff
@@ -207,7 +213,7 @@ void TestCoinsView(FuzzedDataProvider& fuzzed_data_provider, CCoinsView& backend
 
     {
         std::unique_ptr<CCoinsViewCursor> coins_view_cursor = backend_coins_view.Cursor();
-        assert(!coins_view_cursor);
+        assert(is_db ? !!coins_view_cursor : !coins_view_cursor);
```

Comment:
Why explicit conversion with `!!`? Doesn't assert cause contextual conversion to bool of the expression?

E.g.
https://godbolt.org/z/5s9688Thn